packages/ubuntu_bootstrap/lib/l10n/ubuntu_bootstrap_ar.arb

```diff
@@ -15,33 +15,33 @@
   "@quitButtonText": {},
   "welcome": "مرحبا",
   "@welcome": {},
-  "welcomePageTitle": "جرب او ثبت {DISTRO}",
-  "@welcomePageTitle": {},
-  "welcomeRepairDescription": "سيؤدي الإصلاح إلى إعادة تثبيت جميع البرامج المثبتة دون لمس المستندات أو الإعدادات.",
-  "@welcomeRepairDescription": {},
-  "welcomeTryOption": "جرب {RELEASE}",
-  "@welcomeTryOption": {
+  "tryOrInstallPageTitle": "جرب او ثبت {DISTRO}",
+  "@tryOrInstallPageTitle": {},
+  "tryOrInstallRepairDescription": "سيؤدي الإصلاح إلى إعادة تثبيت جميع البرامج المثبتة دون لمس المستندات أو الإعدادات.",
+  "@tryOrInstallRepairDescription": {},
+  "tryOption": "جرب {RELEASE}",
+  "@tryOption": {
     "type": "text",
     "placeholders": {
       "RELEASE": {}
     }
   },
-  "welcomeTryDescription": "يمكنك تجربة {RELEASE} بدون إجراء اي تغييرات لجهازك.",
-  "@welcomeTryDescription": {
+  "tryDescription": "يمكنك تجربة {RELEASE} بدون إجراء اي تغييرات لجهازك.",
+  "@tryDescription": {
     "type": "text",
     "placeholders": {
       "RELEASE": {}
     }
   },
-  "welcomeInstallOption": "ثبت {RELEASE}",
+  "installOption": "ثبت {RELEASE}",
   "@welcomeInstallOption": {
```

Comment:
should be changed to `@installOption` here and probably in all other arb files

Reply:
Good catch!

Reply:
Fixed